core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -449,37 +550,50 @@ object ConsumerGroupCommand extends Logging {
                 }
               }
 
-            val rowsWithoutConsumer = offsets.filterNot {
-              case (topicPartition, offset) => assignedTopicPartitions.contains(topicPartition)
-              }.flatMap {
-                case (topicPartition, offset) =>
-                  val topicAndPartition = new TopicAndPartition(topicPartition)
-                  collectConsumerAssignment(group, Some(consumerGroupSummary.coordinator), Seq(topicAndPartition),
-                      Map(topicAndPartition -> Some(offset)), Some(MISSING_COLUMN_VALUE),
-                      Some(MISSING_COLUMN_VALUE), Some(MISSING_COLUMN_VALUE))
-                }
+            val rowsWithoutConsumer = offsets.filterKeys(!assignedTopicPartitions.contains(_)).flatMap {
+              case (topicPartition, offset) =>
+                val topicAndPartition = new TopicAndPartition(topicPartition)
+                collectConsumerAssignment(group, Some(consumerGroupSummary.coordinator), Seq(topicAndPartition),
+                    Map(topicAndPartition -> Some(offset)), Some(MISSING_COLUMN_VALUE),
+                    Some(MISSING_COLUMN_VALUE), Some(MISSING_COLUMN_VALUE))
+            }
 
             Some(rowsWithConsumer ++ rowsWithoutConsumer)
       }
       )
     }
 
+    override def collectGroupMembers(verbose: Boolean): (Option[String], Option[Seq[MemberAssignmentState]]) = {
+      val group = opts.options.valueOf(opts.groupOpt)
+      val consumerGroupSummary = adminClient.describeConsumerGroup(group, opts.options.valueOf(opts.timeoutMsOpt))
+      (Some(consumerGroupSummary.state),
+        consumerGroupSummary.consumers.map(_.map {
+          consumer => MemberAssignmentState(group, consumer.consumerId, consumer.host, consumer.clientId, consumer.assignment.length,
+            if (verbose) consumer.assignment else List())
+        })
+      )
+    }
+
+    override def collectGroupState(): Option[GroupState] = {
```

Comment:
When debugging a problem, it is often useful to know who the coordinator of a group is. Would it be reasonable to add that to the state output? I don't think we'd need a revote for a minor change like that.

Reply:
Sure, I was going to add it as part of the code change for [KIP-181](https://cwiki.apache.org/confluence/display/KAFKA/KIP-181%3A+DescribeGroups+Protocol+Enhancement) (still in draft mode), but I guess I can update KIP-175 with this minor change and submit a new commit and add coordinator id to to `ConsumerGroupSummary` class.

Reply:
Yeah, I'd suggest just doing it here.

Reply:
Sure, just noticed `coordinator` is already in `ConsumerGroupSummary`. So not much will need to change to get it done.